src/components/document-metadata-editor.js

```diff
@@ -1,153 +0,0 @@
-import PropTypes from 'prop-types';
```

Comment:
Huh! Interesting leftover, this component..

Reply:
Yeah, nice surprise, always happy to delete stuff :-)